README.md

```diff
@@ -15,6 +16,16 @@ depend on the number of vertices, the number of edges, or both?
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. Add your answer to this markdown file.
 
+Sources: Used Google for this assignment
+
+Vertices: The outer loop in the implementation runs V times because it iterates once for each vertex in the graph.
+
+Worst Case: In the worst case scenario, every pair of vertices might have an edge between them in a directed graph. For each of the V vertices, the function checks V possible edges. The complexity is $O(V^2)$.
+
+Best Case: The best case occurs when the number of edges (E) is not the potential maximum in a directed graph. Here we have a best case time complexity $O(V + E)$.
```

Comment:
Don't really see the difference in your code. It looks like you're iterating over the entire matrix in any case.